qek/data/graphs.py

```diff
@@ -36,14 +36,16 @@ class BaseGraph:
 
     device: Final[pl.devices.Device]
 
-    def __init__(self, id: Any, data: pyg_data.Data, device: pl.devices.Device):
+    def __init__(
+        self, id: Any, data: pyg_data.Data, device: pl.devices.Device, target: int | None = None
```

Comment:
Does `id` have to be `Any` or could it be concretised ?

Reply:
Right now, it's an `int`, I can specify it.